src/file_parsing_uq.jl

```diff
@@ -0,0 +1,507 @@
+using TOML
+using Distributions
+using EnsembleKalmanProcesses.ParameterDistributions
+
+
+# "Public" functions (i.e., functions intended to be called by user):
+#    - read_parameters
+#    - get_parameter_distribution
+#    - save_parameter_ensemble
+#    - get_UQ_parameters
+#    - get_regularization
+
+"""
+read_parameters(path_to_toml_file)
+
+Read parameters from toml file
+
+Args:
+`path_to_toml_file` - path of the toml file containing the parameters to be
+                      read.
+                      See `CLIMAParameters/test/uq_test_parameters.toml` for
+                      an example toml file that illustrates the expected
+                      format of the parameter information.
+
+Returns a nested dictionary whose keys are the parameter names (= headers of
+the toml tables) and whose values are dictionaries containing the corresponding
+parameter information (e.g., "prior", "constraint", "value", etc.)
+"""
+function read_parameters(path_to_toml_file::AbstractString)
+    param_dict = TOML.parsefile(path_to_toml_file)
+    return param_dict
+end
```

Comment:
I recommend just inlining, and delete, this method. using `TOML.parsefile(path_to_toml_file)` is self explanatory, and it's a stable interface

Reply:
FWIW, this function was removed from CLIMAParameters

Reply:
Okay, then I will delete the function here too. If e.g. checks/assertions will become necessary at some point we can also do them after reading the parameters in (e.g, placing them in the functions that require the assertions to pass).

Reply:
`read_parameters()` has been deleted.